modules/publication/help/publication.md

```diff
@@ -0,0 +1,16 @@
+# Publication
+
+Publication is intended for users to propose specific research projects
+and for PIs to review them. When proposing a project, PIs may also specify 
+other users to help with editing and maintaining their project proposals.
+
+
+Navigate to the "__Propose a Project__" tab for proposing research projects based on the data required. You may upload publication media, and allows study PIs to review the project 
```

Comment:
The second part of the second sentence "and allows study PIs" isn't grammatically correct. Maybe something like "which may then be reviewed by study PIs" would be better.